src/libstd/time/duration.rs

```diff
@@ -45,6 +45,7 @@ macro_rules! try_opt {
 
 /// ISO 8601 time duration with nanosecond precision.
 /// This also allows for the negative duration; see individual methods for details.
+#[unstable(feature = "std_misc")]
```

Comment:
Ah this was probably a vicitm of inheritance. An `#[unstable]` attribute is inherited by all children by default. The entire `std::time::duration` module is marked `#[unstable]` (which this inherits), but the `hash_map` module is not unstable and `Hasher` was just erroneously missing a marker.